_config.yml

```diff
@@ -32,7 +32,6 @@ google_analytics:
 
 plugins:
   - jekyll-feed
-  - jekyll-redirect-from
```

Comment:
Think this needs to stay

Reply:
I added it back in via a separate commit.  didn't seem to correct the issue though.